extensions/twitter/src/utils.ts

```diff
@@ -4,4 +4,9 @@ export function getErrorMessage(error: unknown): string {
 
 export const padStart = (str: string | number, length: number): string => {
     return String(str).padStart(length, " ");
-};
\ No newline at end of file
+};
+
+export function compactNumberFormat(num: number): string {
+    const fmt = new Intl.NumberFormat('en', { notation: 'compact' });
+    return fmt.format(num)
+}
```

Comment:
You can cache the formatter, otherwise you create a new one with every list item. 

```suggestion

const fmt = new Intl.NumberFormat('en', { notation: 'compact' });

export function compactNumberFormat(num: number): string {
    return fmt.format(num)
}
```